Authentication Methods Registration Campaign Graph Docs (#12112)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/excludetarget.md

---
title: "excludeTarget resource type"

> description: "Users our groups of users to be excluded from a policy."

Reviewer: ```suggestion
description: "Represents the users or groups of users to be excluded from a policy."
```

author: "mjsantani"
localization_priority: Normal

> ms.prod: "microsoft-identity-platform"

Reviewer: ```suggestion
ms.prod: "identity-and-sign-in"
```

doc_type: resourcePageType
---

# excludeTarget resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Determines what users or groups of users are excluded from a particular policy.

Reviewer: ```suggestion
Represents the users or groups of users that are excluded from a policy.
```

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |id|String|Object ID of an Azure AD user or group.|

Reviewer: ```suggestion
|id|String|The object identifier of an Azure AD user or group.|
```

> |targetType|authenticationMethodTargetType|Possible values are: `user`, `group`, `unknownFutureValue`.|

Reviewer: ```suggestion
|targetType|authenticationMethodTargetType|The type of the authentication method target. Possible values are: `user`, `group`, `unknownFutureValue`.|
```
